### ISSUE TITLE
Please allow live map filtering

### ISSUE THREAD

**Issue by mirabilos** [cgeo/cgeo#7278]
Severity: wishlist

Please allow filtering caches in the live map. Most importantly, “hide all saved caches” (so I can download individual unsaved ones from it; this makes spotting them easier, as there are so many caches here that they overlap), but a by-platform filter would also help (to download only one copy of those on OC and GC both).

These filters should be transient, i.e. not saved, and reset when the map is next opened.

**Comment by mirabilos** [comment]
Hm, filtering by cache type is something I imagine people would also want, but not what I described above.

**Comment by SammysHP** [comment]
The mentioned issue is about accessing filters from the live map in general.

**Comment by mirabilos** [comment]
True, but “hide all caches saved offline” is not exactly a regular filter (nor would it make sense there…)

**Comment by SammysHP** [comment]
There is already a filter for stored caches and filters can be inverted.

**Comment by murggel** [comment]
> 
> 
> There is already a filter for stored caches and filters can be inverted.

Hi, I am looking for filtering the caches in the map and so I found this issue. 
@SammysHP: can you tell me, where and how I can filter the caches in the map or is the filter for stored caches only available in the list view?
Normally I have the workflow:
- Open the map
- Show the caches as list
- apply tilter to the list
- open the filtered cache-list in the map (ith option "disable live")
-> now I have the filtered view in the map.
Is there a "shortcut" for this?
